stratum/hal/lib/tdi/tdi_sde_table_entry.cc

```diff
@@ -68,10 +68,14 @@ ::util::Status TdiSdeWrapper::InsertTableEntry(
   tdi_status_t status = table->entryAdd(
       *real_session->tdi_session_, *dev_tgt, flags, *real_table_key->table_key_,
       *real_table_data->table_data_);
-  if (!((status == BF_SUCCESS) || (status == BF_ALREADY_EXISTS)))
+  if ((status == BF_ALREADY_EXISTS)) {
+     return MAKE_ERROR(::util::error::Code::ALREADY_EXISTS) <<
+             "Duplicate table entry with " << dump_args();
+  } else if (!(status == BF_SUCCESS)) {
```

Comment:
`(status != BF_SUCCESS)` would be clearer (and wouldn't require a second set of parentheses)

Reply:
Done